tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -53,7 +53,7 @@ private Security(SecuritySettings settings = null, InstrumentationGateway instru
                 _settings.Enabled = _settings.Enabled && AreArchitectureAndOsSupported();
                 if (_settings.Enabled)
                 {
-                    _powerWaf = powerWaf ?? Waf.Waf.Initialize();
+                    _powerWaf = powerWaf ?? Waf.Waf.Initialize(_settings.Rules);
```

Comment:
This is all in https://github.com/DataDog/dd-trace-dotnet/pull/1757 right? shouldn't be in this pr?

Reply:
Yes, git accident, I was hoping to correct it before anyone noticed, it seems this was not possible 😄